cypress/support/commands.js

```diff
@@ -22,4 +22,16 @@
 //
 //
 // -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
\ No newline at end of file
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+Cypress.Commands.add('checkVoteUpdate', (voteSelector, upvote = true, expectedCount) => {
```

Comment:
This greatly improves the code, looks very clean and follows good convention, excited to look into this.